src/Microsoft.AspNetCore.Blazor.Build.Analyzers/ComponentParametersShouldNotBePublicCodeFixProvider.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Blazor.Build.Analyzers
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(ComponentParametersShouldNotBePublicCodeFixProvider)), Shared]
+    public class ComponentParametersShouldNotBePublicCodeFixProvider : CodeFixProvider
```

Comment:
As you noticed, I'm sure, this won't actually work in .cshtml. It won't be fixed any time soon but we're making progress.

Reply:
Yes. I implemented it anyway because it's cool and quite easy and hopefully one day will work.